project_task_pull_request_state/models/res_config_settings.py

```diff
@@ -0,0 +1,36 @@
+# Copyright Cetmix OU 2023
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl-3.0).
+from odoo import api, fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    pr_state_default = fields.Selection(
+        selection=lambda self: self.env["project.task"]._selection_pr_state(),
+        string="Default PR State",
+        help="Default PR state that will be set when "
+        "PR URI is added to a task in this project",
+    )
+
+    def set_values(self):
```

Comment:
This function is not needed. Please use [config_parameter](https://github.com/odoo/odoo/blob/750d8d939b7e36cf2be810ed450e28827e871a51/addons/account/models/res_config_settings.py#L100) in the field definition instead

```
pr_state_default = fields.Selection(
    selection=lambda self: self.env["project.task"]._selection_pr_state(),
    string="Default PR State",
    config_parameter="project_task_pull_request_state.pr_state_default"
    help="Default PR state that will be set when "
    "PR URI is added to a task in this project",
 )
```